components/Layout/ThemeChanger/ThemeChanger.tsx

```diff
@@ -17,30 +17,24 @@ export const ThemeChanger = () => {
     const toggleTheme = () =>
         theme === 'light' ? setTheme('dark') : setTheme('light');
 
+    const lightClass = theme === 'light' ? styles.light : '';
+
     return (
         <div className={styles.container}>
-            <span
-                className={`${styles.switch_night} ${
-                    theme === 'light' && styles.light
-                }`}
-            >
+            <span className={`${styles.switch_night} ${lightClass}`}>
                 Night
             </span>
-            <label className={styles.switch}>
-                <input
-                    type="checkbox"
-                    onClick={toggleTheme}
-                    defaultChecked={theme === 'light'}
-                />
-                <span className={styles.slider} />
-            </label>
-            <span
-                className={`${styles.switch_day} ${
-                    theme === 'light' && styles.light
-                }`}
+            <button
+                onClick={toggleTheme}
+                className={styles.switch}
+                defaultChecked={theme === 'light'}
+                aria-label={`Switch to ${
+                    theme === 'light' ? 'dark' : 'light'
+                } mode`}
             >
-                Day
-            </span>
+                <span className={`${styles.switch_slider} ${lightClass}`} />
```

Comment:
**Thought** I wonder if these should be read to the screen reader?  might be better to hide them with aria-hidden, might be more of a preference though.

If adding `aria-hidden` is needed, then it would be good to upadte the wagtail.org site also.

Reply:
@Scotchester 
With NVDA, when you focus on the button it reads "switch to light mode. button" and vice versa based on the state - so it reads "switch to dark mode. button" in light mode, as expected.

When using VoiceOver through BrowserStack, it reads "switch to light mode. button" as well, but when you switch between themes it doesnt annouce the state anymore. 

I tried adding `aria-live="polite"` where Ben suggested and it announces the state each time you click the button on VoiceOver, but this isnt helpful for NVDA since when I switch between themes it announces "switch to light/dark mode" 3 times in a row.